### PR TITLE
[Create save datastructure] Load and write level map

### PR DESCRIPTION
**Pull request by patrickrivierejombart**
Load level map example

### CODE REVIEW

File: MAP/level_saves/mapping.yml
Comment (Wolfur1) [inline MAP/level_saves/mapping.yml:1]: Very nice

Comment (Wolfur1) [inline MAP/level_saves/mapping.yml:1]: Very nice

File: MAP/level_saves/starter-village.sav
Comment (Wolfur1) [inline MAP/level_saves/starter-village.sav:1]: Easy to read !

**Review by Wolfur1 — APPROVED** [review]
Good work ! Now you can finaly see your family again.

**Review by TendoYuki — APPROVED** [review]

**Pull request merged**